ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V6__activate_template_id_interning.sql

```diff
@@ -0,0 +1,19 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+DROP INDEX participant_events_divulgence_template_id_idx;
+DROP INDEX participant_events_create_template_id_idx;
+DROP INDEX participant_events_consuming_exercise_template_id_idx;
+DROP INDEX participant_events_non_consuming_exercise_template_id_idx;
+
+ALTER TABLE participant_events_divulgence
+  MODIFY template_id NUMBER;
```

Comment:
This will likely fail if the table is not empty (conversion from string to integer). Which is fine because we don't support data continuity for Oracle yet.